build/template.runsettingsproj

```diff
@@ -49,6 +49,7 @@
 
   <ItemGroup Label="Test Containers">
     <TestContainer Include="$(FileName)" />
+    <TestContainer Include="NuGet.Tests.Apex.dll" Condition=" '$(RunName)' == 'NuGet.Tests.Apex.Daily' " />  
```

Comment:
Can this be passed from runsettings.proj instead?

FileName is our own property, so we can make different, like for example, we can split by `,` or something and include all of that as a test container.

Reply:
> Can this be passed from runsettings.proj instead?
> 
> FileName is our own property, so we can make different, like for example, we can split by `,` or something and include all of that as a test container.

@nkolev92 I tried split the multiple .dlls with `&` and ` ,` respectively and assign it to FileName, the run failed at `Deploy Tests Install-TestContainers` . 
![image](https://github.com/NuGet/NuGet.Client/assets/149561645/0df83d6e-cfb9-4ef4-83fd-88b7e550033e)
I'm not sure if a test container can include mutiple dlls. Do you have any suggesstion or idea?

Reply:
MSBuild syntax is to use a semicolon: https://learn.microsoft.com/en-us/visualstudio/msbuild/how-to-select-the-files-to-build?view=vs-2022#to-declare-multiple-items

Reply:
@zivkan @nkolev92 Thanks for your guidance. Updated the code with your suggestion.

Reply:
Looking at the daily test pipeline, I don't see a manual run of this branch to validate that you've tested this works.  Have you tested this change somewhere else?

Reply:
> @SueSu01 that CI run looks like it only ran the apex tests, but not the daily tests assembly. So, it only ran one of the two test assemblies.

Currently daily tests include 42 P0 and P1 apex cases. The NuGet.Tests.Apex.dll assembly has 101 apex tests and I checked that CI run has 143 tests, so it ran two test assemblies.

Reply:
![image](https://github.com/NuGet/NuGet.Client/assets/2878341/26893c59-6556-49fc-bc23-683f8ada4d53)

This shows the multiple assemblies were run too. 

Conveniently there was a failure in each :D

Reply:
Yes, we will try to investigate the failures in the recent runs during the following week.